subsys/lorawan/lorawan.c

```diff
@@ -472,6 +495,26 @@ int lorawan_set_class(enum lorawan_class dev_class)
 	return 0;
 }
 
+int lorawan_set_channels_mask(uint16_t *channels_mask, uint8_t channels_mask_size)
```

Comment:
Please add some tests to cover this function. The current test suite seems to have that untouched.

Reply:
Fixed.